ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -512,9 +512,13 @@ def pytest_runtest_makereport(item, call):
     if rep.failed and ocsci_config.RUN.get("cli_params").get("collect-logs"):
         test_case_name = item.name
         ocp_logs_collection = True if rep.when == "call" else False
-        mcg = (
-            True if any(x in item.location[0] for x in ["mcg", "ecosystem"]) else False
-        )
+        mcg = False
+        if float(ocsci_config.ENV_DATA["ocs_version"]) < 4.7:
```

Comment:
Do we have a workaround issue created for rolling this back once we can continue to collect noobaa db dump for 4.7?

Reply:
good point - https://github.com/red-hat-storage/ocs-ci/issues/3525